e2e/src/components/ButtonWriteToFile.tsx

```diff
@@ -0,0 +1,27 @@
+import tutorialStore from 'tutorialkit:store';
+
+interface Props {
+  filePath: string;
+  newContent: string;
+  testId?: string;
+}
+
+export function ButtonWriteToFile({ filePath, newContent, testId = 'write-to-file' }: Props) {
```

Comment:
Nice! I can imagine this utility component being useful in other future tests as well. It's good to have it accepting props like this. 🎉